packages/@glimmer/manager/lib/internal/index.ts

```diff
@@ -217,7 +217,7 @@ export function hasInternalComponentManager(definition: object): boolean {
 }
 
 export function hasInternalHelperManager(definition: object): boolean {
-  return getManager(HELPER_MANAGERS, definition) !== undefined;
+  return Boolean(getInternalHelperManager(definition, true));
```

Comment:
Hm, I don't think we can do this exactly, for one thing `getInternalHelperManager` would throw errors which would be problematic. Maybe we add something like this:

```ts
function hasDefaultManager(map: WeakMap<object, object>, definition: object): boolean {
  if (FEATURE_DEFAULT_HELPER_MANAGER) {
    if (map === HELPER_MANAGERS) {
      return typeof object === 'function';
    }
  }
  
  return false;
}
```

Then each of these `has*` functions can be updated to `hasDefaultManager(..., definition) || getManager(..., definition) !== undefined` and we can update this when we add other default managers.

I think it is safe (and faster) to do the default check first in all cases since you can't "unset" the default manager as far as I know, so if it has a default manager, then regardless of whether it had a more specific one, the answer is still going to be true.

Does that seem good to you?

Reply:
> Then each of these has* functions can be updated to ...

I wasn't sure if you meant that I should do that as part of this PR so I added it like that just in case. :smile:  Let me know if it should only be added to the helper version for now.